src/cleaner/config/cleaner.py

```diff
@@ -18,23 +18,29 @@
 
 import logging
 import logging.config
+import copy
 
+class Cleaner(object):
 
-class Cleaner:
-
-    def __init__(self, cluster_configuration, service_configuration, default_service_configuraiton):
+    def __init__(self, cluster_conf, service_conf, default_service_conf):
         self.logger = logging.getLogger(__name__)
-
-        self.cluster_configuration = cluster_configuration
+        self.cluster_conf = cluster_conf
+        self.service_conf = service_conf
+        self.default_service_conf = default_service_conf
 
     def validation_pre(self):
         return True, None
 
     def run(self):
-        com = {}
-
-        return com
-
-    def validation_post(self, cluster_object_model):
+        result = copy.deepcopy(self.default_service_conf)
+        result.update(self.service_conf)
+        return result
+
+    def validation_post(self, conf):
+        threshold = conf["cleaner"].get("threshold")
+        if type(threshold) != int:
```

Comment:
might convert it to int in a `try ... catch`, then check whether it's in `(0, 100)` is better.